pkgs/tools/admin/hop-cli/default.nix

```diff
@@ -10,16 +10,22 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "hop-cli";
-  version = "0.2.35";
+  version = "0.2.52";
 
   src = fetchFromGitHub {
     owner = "hopinc";
     repo = "cli";
     rev = "v${version}";
-    hash = "sha256-TgPEcsv7/n+PzZXazozbgmX2tt4WDvyH3j6rY+M0AGE=";
+    hash = "sha256-xuOkW5jetl+8obeFJnbkVZa+wYWfTNiTOmcrzC8+wGE=";
   };
 
-  cargoHash = "sha256-HEUsyboZQ4j5IEOqiWEBSlJqmaNDHPg1kstnM9AgZBo=";
+  cargoHash = "sha256-ePUlw4UzsQ2lNuJ5g5OAYh6nKTIoHdDMb34Jzuqtas8=";
+
+  postPatch = ''
+      # This test fails on read-only filesystems
+      substituteInPlace src/commands/volumes/utils.rs \
+        --replace '#[tokio::test]' '#[tokio::test] #[ignore]'
+    '';
```

Comment:
instead of patching the file you should do:
```nix
rustPlatform.buildRustPackage {
  /* ... */
  checkFlags = [
    # reason for disabling test
    "--skip=example::tests:example_test"
  ];
}
```

like described in [Rust Disabling package-tests](https://nixos.org/manual/nixpkgs/stable/#rust)